cli/test/kots_app_test.go

```diff
@@ -93,34 +93,13 @@ var _ = Describe("kots apps", func() {
 			req.NotEmpty(stdout.String(), "Expected stdout output")
 
 			req.Equal(stdout.String(),
-`ID                             NAME           SCHEDULER
-`+ app.ID +`    ` + app.Name + `    kots
+`ID                             NAME           SLUG           SCHEDULER
+`+ app.ID +`    ` + app.Name + `    ` + app.Slug + `    kots
 `)
 		})
 	})
-	Context("replicated app create", func() {
-		It("should create an app", func() {
-			newName := mustToken(8)
-			var stdout bytes.Buffer
-			var stderr bytes.Buffer
-
-			rootCmd := cmd.GetRootCmd()
-			rootCmd.SetArgs([]string{"app", "create", newName})
-
-			err = cmd.Execute(rootCmd, nil, &stdout, &stderr)
-			req.NoError(err)
-
-			req.Empty(stderr.String(), "Expected no stderr output")
```

Comment:
IMO these need to get included after the 'app create' below

Reply:
good call thank you